nservicebus/hosting/index_diagnostics_core_[7,).partial.md

```diff
@@ -0,0 +1,25 @@
+## Startup diagnostics
+
+To make troubleshooting easier diagnostics is collected during enpoint startup and written to a `.diagnostics` subfolder in the host output directory.
```

Comment:
I think this sentence misses a `,`

Reply:
also, is that correct: "diagnostics is collected"? Should that be something like "diagnostic data is collected"?